.travis.yml

```diff
@@ -7,6 +7,6 @@ os:
   - osx
 
 node_js:
-  - 7
+  - 10
+  - 8
   - 6
-  - 4
```

Comment:
Yargs version 12 dropped official support for Node 4:

https://github.com/yargs/yargs/blob/master/CHANGELOG.md#breaking-changes-1